src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.OSX.cs

```diff
@@ -13,6 +13,7 @@
 using FSEventStreamEventId = System.UInt64;
 using CFRunLoopRef = System.IntPtr;
 using Microsoft.Win32.SafeHandles;
+using static Interop.EventStream;
```

Comment:
FWIW, we decided elsewhere not to do this:
https://github.com/dotnet/corefx/issues/6408